security/ssh-askpass/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        theseal ssh-askpass 1.4.0 v
+github.tarball_from archive
+revision            0
+
+categories          security devel
+license             ISC
+maintainers         {judaew @judaew}
```

Comment:
You're restricting the maintenance of this port to only yourself intentionally, correct?